eng/CredScanSuppression.json

```diff
@@ -1,4 +1,21 @@
 {
-    "tool": "Credential Scanner",
-    "suppressions": []
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "hash": "efQ8O5wj+98pO9uoDas4uhlPwdp5GHCRs9N2Da2EC/g=",
```

Comment:
If these secrets are the same fake secret there is a way to suppress the pattern instead of all the instances.